atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/SweepTaskRunner.java

```diff
@@ -179,8 +179,8 @@ private SweepResults doRun(TableReference tableRef,
                 lastRow = batch.lastCellExamined().getRowName();
             }
             return SweepResults.builder()
-                    .previousStartRow(startRow)
-                    .nextStartRow(Arrays.equals(startRow, lastRow) ? Optional.absent() : Optional.of(lastRow))
+                    .previousStartRow(Optional.of(startRow))
```

Comment:
Searched for `previousStartRow` internally and it doesn't seem like anyone else directly manipulates `SweepResults` (as you might expect), so this change looks safe.